osf_tests/test_search_views.py

```diff
@@ -95,6 +95,80 @@ def test_search_projects(self):
         res = self.app.get(url, {'q': self.project.title})
         assert_equal(res.status_code, 200)
 
+    def test_search_user(self):
+
+        url = '/api/v1/search/user/'
```

Comment:
If we check the screenshots on the original ticket, they show what look to be global search page results.